shell-ui/src/NavBar.js

```diff
@@ -2,64 +2,95 @@
+      .filter((entry: [string, TranslationAndGroups]) =>
+        isEntryAccessibleByTheUser(entry, userGroups),
+      )
+      .map(
+        //$FlowIssue - flow typing for Object.entries incorrectly typing values as [string, mixed] instead of [string, TranslationAndGroups]
+        ([path, translationAndGroup]: [string, TranslationAndGroups], i) => {
+          try {
+            return {
+              ...renderer(path, translationAndGroup),
+              selected: normalizedLocation === normalizePath(path),
+            };
+          } catch (e) {
+            throw new Error(
+              `[navbar][config] Options path should be provided thanks to fully qualified urls such as {protocol}://{host}{path}?{queryParams} but got "${path}" in ${entries}[${i}] `,
+            );
+          }
+        },
+      )
+  );
+};
+
+export const Navbar = ({ options }: { options: Options }): Node => {
   const auth = useAuth();
 
-  const tabs = [
-    {
-      selected: true,
-      title: 'Overview',
-      onClick: () => console.log('Overview'),
-    },
-  ];
+  const userGroups: string[] = auth.userData?.profile?.groups || [];
+  const accessiblePaths = getAccessiblePathsFromOptions(options, userGroups);
+  useLayoutEffect(() => {
+    accessiblePaths.forEach(accessiblePath => {
+      prefetch(accessiblePath);
+    });
+  }, [JSON.stringify(accessiblePaths)]);
+
+  const tabs = translateOptionsToMenu(
+    options,
+    'main',
+    (path, translationAndGroup) => ({
+      link: <a href={path}>{translationAndGroup.en}</a>,
+    }),
+    userGroups,
+  );
+
   const rightActions = [
-    {
-      type: 'dropdown',
-      text: 'FR',
-      icon: <i className="fas fa-globe" />,
-      items: [
-        {
-          label: 'English',
-          name: 'EN',
-          onClick: () => console.log('en'),
-        },
-      ],
-    },
-    {
-      type: 'dropdown',
-      icon: <i className="fas fa-question-circle" />,
-      items: [
-        { label: 'About', onClick: () => console.log('About clicked') },
-        {
-          label: 'Documentation',
-          onClick: () => console.log('Documentation clicked'),
-        },
-        {
-          label: 'Onboarding',
-          onClick: () => console.log('Onboarding clicked'),
-        },
-      ],
-    },
-    {
-      type: 'button',
-      icon: <i className="fas fa-sun" />,
-      onClick: () => console.log('Theme toggle clicked'),
-    },
     {
       type: 'dropdown',
       text: auth.userData?.profile.name || '',
       icon: <i className="fas fa-user" />,
       items: [
+        ...translateOptionsToMenu(
+          options,
+          'subLogin',
+          (path, translationAndGroup) => ({
+            label: translationAndGroup.en,
+            onClick: () => {
+              location.href = path;
+            },
+          }),
+          userGroups,
+        ),
         {
           label: 'Log out',
```

Comment:
I'm wondering (that's likely just a design question): do we want the `Log out` entry to always be the last? I'd instinctively put it first, but maybe my instincts are wrong :sweat_smile:

Reply:
Github put it at the end : 
![image](https://user-images.githubusercontent.com/75977494/110928954-ba731700-8327-11eb-8860-5e2383b92ea4.png)

@Cuervino do you have an opinion on this ?

Reply:
I checked on few apps: on most of them, the "Log out" option is at the last position of the menu list.
Having it on the last position is also nice for avoiding "Accidental logouts" (harder to misclick when the "log out" is further from the first click on the icon for accessing the menu).
So for these 2 reasons (**Avoid Accidental logouts** and **Follow standards**), my opinion is we should keep it at the last position.